### PR TITLE
Remove `events` and `rejections` options

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR removes `events` and `rejections` Spine options as Model entries should describe entity state rather than behavior.

The options are used in `users` project but there is no evidence that anyone is parsing them, so the removal shouldn't introduce any errors.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@armiol PTAL.

**Comment by alexander-yevsyukov** [comment]
@armiol, until this PR we used to speak about messages as entity states. Now with these options they also describe behavior. It's sort of confusing. Let's discuss.

File: base/src/main/java/io/spine/code/proto/MessageType.java [hunk 1]
```diff
@@ -64,6 +65,15 @@ public MessageType(Descriptor descriptor) {
         super(descriptor, true);
     }
 
+    /**
+     * Creates a new instance from the given {@code Message} class.
+     */
+    public static MessageType of(Class<? extends Message> aClass) {
```
Comment (alexander-yevsyukov) [inline base/src/main/java/io/spine/code/proto/MessageType.java]: Please have a constructor instead. We have types that derive from `MessageType`. Such a static method may be a dis-service.

**Comment by alexander-yevsyukov** [comment]
@dmitrykuzmin, I relation to my comment regarding the factory method of `MessageType`, there are some changes to the type-related classes that I'm currently working on. See #340.

**Comment by armiol** [comment]
@alexander-yevsyukov the reason we speak of `lifecycle` is the same as speaking of which `(events)` are emitted in response to which commands. Let's discuss vocally.

**Comment by dmitrykuzmin** [comment]
@armiol PTAL again.

**Pull request merged**